.github/workflows/all-checks.yml

```diff
@@ -0,0 +1,44 @@
+name: Run all checks on Kedro-Viz
+on:
+  workflow_call:
+  workflow_dispatch:
```

Comment:
Can actually just add -
```
schedule:
  - cron: 0 1 * * *
 ```
And get rid of `daily.yml`

Reply:
Okay, like this
```
on:
  workflow_call:
  workflow_dispatch:
  schedule:
    - cron: 0 1 * * *  
```

So all-checks.yml will work as standalone and reusable workflow right?

Reply:
Yeah, because daily.yml doesn't have any other thing except for calling this workflow!